kernel/data-pipeline/dialect/mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/PasswordEncryption.java

```diff
@@ -99,15 +99,15 @@ private static byte[] encryptWithRSAPublicKey(final byte[] source, final RSAPubl
     }
     
     private static RSAPublicKey parseRSAPublicKey(final String key) throws GeneralSecurityException {
-        byte[] certificateData = Base64.getDecoder().decode(formatKey(key));
+        byte[] certificateData = Base64.getDecoder().decode(formatKey(key.trim()));
```

Comment:
It's better to put `key.trim()` in `formatKey` method, since it's the duty of `formatKey`